media_daemon/DAEMON.md

```diff
@@ -20,4 +20,4 @@ We are using:
 
 ## Workflow
 
-<img src="https://raw.github.com/fossasia/susi_linux/docs/images/media_daemon.svg">
\ No newline at end of file
+<img src="https://raw.github.com/fossasia/susi_linux/docs/images/media_daemon.svg">
```

Comment:
The svg is not showing. 
Replace by

`https://raw.github.com/fossasia/susi_linux/master/docs/images/media_daemon.svg`